### DISCUSSION TITLE
Trying to run e-Dokyumento on my Ubuntu laptop

### DISCUSSION THREAD

**Discussion by Saiks1** [community/community#69882] [Programming Help]
### Select Topic Area

Product Feedback

### Body

I need help on this!!!
![Screenshot from 2023-10-12 14-32-08](https://github.com/community/community/assets/103122643/13f8fa91-9426-4b0b-87a0-075d253fd816)

**Comment by nelsonmaligro** [comment]
You need to import the JSON files into the mongodb

**Comment by Unmesh100** [comment]
It looks like you're trying to run e-Dokyumento on your Ubuntu laptop and encountered an issue, but the description of the problem is missing crucial details that would help others troubleshoot the issue effectively.

To get the help you need, consider providing the following information:

Error Message: What error or issue are you encountering? If there's a specific error message, share it verbatim.
Steps to Reproduce: Describe how you're trying to run e-Dokyumento—commands you're using, and what you did before the issue occurred.
System Information: Which version of Ubuntu are you using?
Screenshot Details: If you mentioned a screenshot, make sure it clearly shows the issue or error.
